packages/web-api/src/response/AdminAnalyticsGetFileResponse.ts

```diff
@@ -0,0 +1,8 @@
+/* tslint:disable */
```

Comment:
Some parts of auto-generated code are not compatible with the current tslint settings. I tried to settle all of them [this way](https://github.com/slackapi/node-slack-sdk/blob/94eb35ad1361a52f65ac43a289b99e703810827f/scripts/code_generator.rb#L24-L26) but a few patterns are inevitable. You can see the result by modifying the code generator.